modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,184 @@
+* You have more than one physical interfaces attached to your primary network and you defined the interfaces in a NAD CRD file.
+
+.Procedure
+
+. For each bare-metal host that exists in a cluster, in the `install-config.yaml` file for your cluster define a `networkConfig` section similar to the following example:
++
+[source,yaml]
+----
+# ...
+networkConfig:
+  interfaces:
+    - name: enp1s0 <1>
+      type: ethernet
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        enabled: false
+    - name: enp2s0 <2>
+      type: ethernet
+      state: up
+      mtu: 1500 <3>
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+    - name: enp3s0 <4>
+      type: ethernet
+      state: up
+      mtu: 1500
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+# ...
+----
+<1> The interface for the provisioned network interface controller (NIC). 
+<2> The first bonded interface that pulls in the Ignition config file for the bond interface.
+<3> Manually set the `br-ex` MTU on the bond ports.
+<4> The second bonded interface is part of a minimal configuration that pulls ignition during cluster installation.
+
+. Define each network interface in an NMState configuration file:
++
+.Example NMState configuration file that defines many network interfaces
+[source,yaml]
+----
+ovn:
+  bridge-mappings:
+    - localnet: localnet-network
+      bridge: br-ex
+      state: present
+interfaces:
+  - name: br-ex
+    type: ovs-bridge
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false
+    ipv6:
+      enabled: false
+      dhcp: false
+    bridge:
+      allow-extra-patch-ports: true
+      port:
+        - name: br-ex
+        - name: patch-ex-to-phy
+    ovs-db:
+      external_ids:
+        bridge-uplink: "patch-ex-to-phy"
+  - name: br-ex
+    type: ovs-interface
+    state: up
+    mtu: 1500 <1>
+    ipv4:
+      enabled: true
+      dhcp: true
+      auto-route-metric: 48
+    ipv6:
+      enabled: false
+      dhcp: false
+      auto-route-metric: 48
+  - name: br-phy
+    type: ovs-bridge
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false
+    ipv6:
+      enabled: false
+      dhcp: false
+    bridge:
+      allow-extra-patch-ports: true
+      port:
+        - name: patch-phy-to-ex
+        - name: ovs-bond
+          link-aggregation:
+            mode: balance-slb
+            port:
+              - name: enp2s0
+              - name: enp3s0
+  - name: patch-ex-to-phy
+    type: ovs-interface
+    state: up
+    patch:
+      peer: patch-phy-to-ex
+  - name: patch-phy-to-ex
+    type: ovs-interface
+    state: up
+    patch:
+      peer: patch-ex-to-phy
+# ...
+----
+<1>
+
+. Use the `cat` command to base64-encode the interface content of the NMState configuration file:
++
+[source,terminal]
+----
+$ cat <nmstate_configuration>.yaml | base64 -w0 <1>
```

Comment:
To be consistent with the day-2 docs, use `.yml` suffix and simplify `base64` command.

```shell
$ base64 -w0  <nmstate_configuration>.yml
```

Also remove the `cat` line above.